app/src/main/java/co/nimblehq/alpine/sample/extension/ViewModelExt.kt

```diff
@@ -0,0 +1,20 @@
+package co.nimblehq.alpine.sample.extension
+
+import androidx.activity.ComponentActivity
+import androidx.activity.viewModels
+import androidx.annotation.MainThread
+import androidx.lifecycle.*
+import androidx.lifecycle.viewmodel.CreationExtras
+
+@MainThread
+inline fun <reified VM : ViewModel> ComponentActivity.provideViewModels(
+    noinline factoryProducer: (() -> CreationExtras)? = null
+): Lazy<VM> = OverridableLazy(viewModels(factoryProducer))
```

Comment:
Are we planning to include UI tests here as well? 🤔

Reply:
@ryan-conway I added this extension because for using when initializing `ViewModel` in `Activity` as we call in `CameraCaptureActivity` at line `39`, but for `UI Test` and `UnitTest`, I think we can't include them for this round 🙏